libs/unity/library/Runtime/Scripts/NativeRender/NativeVideoRenderer.cs

```diff
@@ -24,8 +26,9 @@ public class NativeVideoRenderer : MonoBehaviour
         private Texture2D _textureU;
         private Texture2D _textureV;
 
-        private VideoFrameQueue<I420AVideoFrameStorage> _i420aFrameQueue;
-
+        private int dirtyWidth;
+        private int dirtyHeight;
```

Comment:
Should use the `_` prefix for private fields, so `_dirtyWidth`, `_dirtyHeight`. Also might benefit from a comment saying what they are, "dirty" implies that it's used when res changes but it's not clear if this is the old res or the new res.